### PR TITLE
update $to modifier compatibility

### PR DESCRIPTION
**Pull request by stanislav-atr**

### CODE REVIEW

**Review by ameshkov — CHANGES_REQUESTED** [review]
There are some major changes in the document formatting, please merge with master

File: docs/general/ad-filtering/create-own-filters.md
Comment (maximtop) [inline docs/general/ad-filtering/create-own-filters.md:539]: Is Opera intentionally omitted?

Comment (stanislav-atr) [reply docs/general/ad-filtering/create-own-filters.md:539]: i got that from $domain modifier compatibility string. Opera is also chromium-based

Comment (ameshkov) [reply docs/general/ad-filtering/create-own-filters.md:539]: Chromium and Firefox?

Comment (slavaleleka) [reply docs/general/ad-filtering/create-own-filters.md:539]: maybe no need to specify browsers since all of them are supported? I think `AdGuard Browser extension **running ...` is enough

Comment (el-termikael) [reply docs/general/ad-filtering/create-own-filters.md:539]:
```suggestion
Rules with the `$to` modifier are supported by AdGuard for Windows, Mac, and Android, **running CoreLibs v1.12 or later**, and AdGuard Browser Extension, **running TSUrlFilter v2.1.3 or later**.
```

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by sxgunchenko — APPROVED** [review]

**Comment by stanislav-atr** [comment]
> There are some major changes in the document formatting, please merge with master

@ameshkov 
i've merged those changes, could you please approve this, so i can merge after @el-termikael approves too?

**Review by slavaleleka — APPROVED** [review]

**Review by el-termikael — APPROVED** [review]

**Pull request merged**
